ci/atlas_published_likelihoods.py

```diff
@@ -0,0 +1,32 @@
+from selenium import webdriver
+
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.options import Options
+
+url = 'https://twiki.cern.ch/twiki/bin/view/AtlasPublic'
+
+service = Service('/usr/bin/chromedriver')
+
+options = Options()
+options.add_argument("--headless")
+options.add_argument('--no-sandbox')
+options.add_argument('--disable-gpu')
+options.add_argument('--disable-dev-shm-usage')
+options.add_argument('disable-infobars')
+options.add_argument("--disable-extensions")
+
+with webdriver.Chrome(options=options, service=service) as driver:
+    driver.get(url)
+    driver.execute_script("addKeyword('Analysischaracteristics_Likelihood@available');")
+    rows = driver.find_elements('css selector', '#paperListTbody tr')
+
+    for row in rows:
```

Comment:
Maybe we could enumerate this so that in addition to the entries we can easily spot check the number of entries.